mscore/preferences.cpp

```diff
@@ -1149,6 +1151,18 @@ void PreferenceDialog::resetShortcutClicked()
       active->setText(1, shortcut->keysToString());
       shortcutsChanged = true;
       }
+    
+void PreferenceDialog::saveShortcutListClicked()
+      {
+      QString saveFileName = QFileDialog::getSaveFileName(new QWidget(), "Save Shortcuts", "", tr("XML Files(*.xml)"));
+      Shortcut::saveToNewFile(saveFileName);
+      }
+    
+void PreferenceDialog::loadShortcutListClicked()
+      {
+      QString loadFileName = QFileDialog::getOpenFileName(new QWidget(), "Load Shorcuts", "", tr("XML Files(*.xml)"));
```

Comment:
Also please compare the other places where we load and save files, esp. if `preferences.nativeDialogs` is false (like on Linux I believe), as it had been mentioned on IRC yestertday